src/distributions/other.rs

```diff
@@ -62,6 +62,10 @@ impl Distribution<char> for Alphanumeric {
             b"ABCDEFGHIJKLMNOPQRSTUVWXYZ\
                 abcdefghijklmnopqrstuvwxyz\
                 0123456789";
+        // We can pick from 62 characters. This is so close to a power of 2, 64,
+        // that we can do better than `Range`. Use a simple bitshift and
+        // rejection sampling. We do not use a bitmask, because for small RNGs
+        // the most significant bits are usually of higher quality.
         loop {
             let var = rng.next_u32() >> 26;
```

Comment:
Maybe write `(32 - 6)` so people don't get the wrong idea about where 26 comes from?